cfn/handler/request.go

```diff
@@ -35,9 +53,15 @@ type Request struct {
 }
 
 // NewRequest returns a new Request based on the provided parameters
-func NewRequest(id string, ctx map[string]interface{}, sess *session.Session, previousBody, body []byte) Request {
+func NewRequest(id string, NextToken string, StackId string, StackTags map[string]string, Region string, AccountID string, SystemTags map[string]string, ctx map[string]interface{}, sess *session.Session, previousBody, body []byte) Request {
```

Comment:
Same thing here, we need a better way to pass in this information. I suggest creating a new type.